libtransmission/peer-mgr-wishlist.cc

```diff
@@ -370,7 +371,19 @@ class Wishlist::Impl
                     return 1U;
                 }
 
-                return piece + 1U;
+                if (sequential_download_from_piece == 0)
```

Comment:
Very small improvement to the execution path.

```suggestion
                if (sequential_download_from_piece <= 1)
```